neqo-crypto/src/agent.rs

```diff
@@ -686,14 +686,20 @@ impl ::std::fmt::Display for SecretAgent {
     }
 }
 
+#[derive(Debug, PartialOrd, Ord, PartialEq, Eq, Clone)]
+pub struct ResumptionToken {
+    pub token: Vec<u8>,
+    pub expiration_time: i64,
+}
```

Comment:
We can probably implement `AsRef<[u8]>` for this to get at the contents of the token more easily.  And modify those functions that take a token to take `impl AsRef<[u8]>` instead of `&[u8]`.  That's ergonomics and should avoid needing to make the fields public.

For the expiration time, this needs to be an `Instant`, not `PRTime` and I would still prefer to use an accessor.  We should not be exposing the internal structure of objects.